README.md

```diff
@@ -66,8 +66,10 @@ The end-to-end tests are powered by [nightmare](http://nightmarejs.org/),
 **Test Running Command**
 
 ```shell
-$ ./node_modules/mocha/bin/mocha test/end-to-end/test                # Run all of end-to-end test cases
-$ ./node_modules/mocha/bin/mocha test/end-to-end/test/test_xyz.js    # Run test cases in test_xyz.js only
+$ cd test/end-to-end
+$ npm install                                   # Install end-to-end dependencies
+$ npm run test                                  # Run end-to-end test
+$ npm run xunittest                             # Run end-to-end test with xunit xml result output
 ```
```

Comment:
No need to duplicate this here. You can just refer to the appropriate readme for the tests.